.github/workflows/build.yml

```diff
@@ -1,9 +1,8 @@
+# @format
```

Comment:
What does this do? :thinking:

Reply:
Nice catch. I hadn't thought about this, but prettier inserts this if you have insertPragma set to true: https://stackoverflow.com/questions/67358902/in-vs-code-this-line-format-always-appears-at-the-top-of-my-js-files-on

I removed it from my settings.